### PR TITLE
Small update to instructional comments in app template

### LINKED ISSUE #7990: Starter app's source code has hot reload instructions only for terminal, doesn't mention our IDE
From a UX study:

The comments/instructions for Hot Reload in the starter app's source code (after "new ThemeData") are written for someone using the terminal. Do we need to also add IntelliJ instructions here?

### PR DESCRIPTION
**Pull request by mit-mit**
- Add IntelliJ as an option under reload

Fixes: https://github.com/flutter/flutter/issues/7990

- Make a few comments fit one line

### CODE REVIEW

**Comment by mit-mit** [comment]
@Hixie as discussed in #7990, I think this is subtle enough that it fits with our IDE being recommended.

**Comment by mit-mit** [comment]
@sethladd please review

File: packages/flutter_tools/templates/create/lib/main.dart.tmpl [hunk 1]
```diff
@@ -5,8 +5,7 @@ import 'package:flutter_driver/driver_extension.dart';
 
 void main() {
 {{#withDriverTest}}
-  // Starts the app with Flutter Driver extension enabled to allow Flutter Driver
-  // to test the app.
+  // Starts with Flutter Driver extension enabled allowing it to test the app.
```
Comment (sethladd) [inline packages/flutter_tools/templates/create/lib/main.dart.tmpl]: Maybe: "Allow the Flutter Driver to test the app. See goo.gl/i83h2f for more info."

(I made up that link. Maybe point it to https://flutter.io/testing/ ?)

**Review by sethladd — COMMENTED** [review]
other tweaks lgtm. Thanks!

File: packages/flutter_tools/templates/create/lib/main.dart.tmpl (hunk 1 above)
Comment (Hixie) [inline packages/flutter_tools/templates/create/lib/main.dart.tmpl]: I don't really have a suggestion but I agree with Seth that the current text and the proposed text are both suboptimal here.

**Comment by Hixie** [comment]
I think this is fine, yeah. LGTM modulo seth's comment.

**Comment by mit-mit** [comment]
Driver comment updated, PTAL!

**Comment by mit-mit** [comment]
@sethladd can you take a final look?

**Comment by Hixie** [comment]
LGTM

**Pull request merged**